jupyterhub/spawner.py

```diff
@@ -944,15 +944,13 @@ async def move_certs(self, paths):
 
         Arguments:
             paths (dict): a list of paths for key, cert, and CA.
-            These paths will be resolvable and readable by the Hub process,
-            but not necessarily by the notebook server.
+            These paths will be resolvable and readable by the Hub process, but
+            not necessarily by the notebook server.
 
         Returns:
-            dict: a list (potentially altered) of paths for key, cert,
-            and CA.
-            These paths should be resolvable and readable
-            by the notebook server to be launched.
-
+            dict: a list (potentially altered) of paths for key, cert, and CA.
+            These paths should be resolvable and readable by the notebook
+            server to be launched.
```

Comment:
There's a conflict on these changes, but it looks like there aren't any intentional changes here. Rebasing without the changes to spawner.py should fix the conflict